src/niscope/metadata/functions.py

```diff
@@ -1293,7 +1293,7 @@
                 },
                 'name': 'timeout',
                 'python_api_converter_name': 'convert_timedelta_to_seconds_real64',
-                'python_type': 'float or hightime.timedelta',
+                'python_type': 'hightime.timedelta, datetime.timedelta, or float',
```

Comment:
It looks like this change didn't cause any change in generated code. Do you know why?

Reply:
Not sure. The generated code in this commit is up to date. I'm not sure if it matters whether it changed generated code, it's now consistent with every other instance of python_type referencing datetime/hightime in the functions.py file.

Reply:
It's probably because we had overrides in the add-ons. They are now redundant and should be removed in this same PR. Please go through the add-on metadata and remove anything that is no longer needed.

Reply:
There's nothing in the add-on files that would override the FancyFetch function.

Reply:
I meant in general to look for any overrides that are rendered redundant by updating source metadata.

Reply:
Got it. I think there was some confusion on my side, I wasn't integrating functions that were in the addon file into the source metadata, but I should be able to get that done pretty quickly. I'll make a perforce change to reflect that and then remove the redundant addon information.

Reply:
Done. functions_addon.py is clean.

Reply:
Actually, the answer to my original question is.. `type_in_documentation` key overrides `python_type` key. For parameters that use converters, `python_type` is not required. You can remove that key (both here and in internal metadata)

Marcos' comment about overrides in add-ons was about the `functions_override_metadata` dict in addon file. `functions_additional_..` dicts in addon file are required (since we have fancy templates for them). If you delete them and rebuild, fancy methods will not be codegen'd in session.py.

Reply:
Actually, it looks like you moved the metadata out of addon file, rather than delete them. Keep them in the addon file.

Reply:
Ignore the 2nd part of the previous comment. You've moved the fancy metadata, which is fine.

Reply:
Good to know, I'll remove the redundancy. I'd like to suggest ignoring it for this PR since it doesn't break anything and the redundancy already existed in the codebase so that it can be grouped in with the next set of perforce changes I make in order to complete #1485. Would this be acceptable or should it be removed immediately?

Reply:
That's fine. A separate PR for just this change would also be ok.

Reply:
Preferable, even. Smaller, incremental, self contained PRs FTW